postCore/src/main/java/com/hamdan/forzenbook/post/core/domain/PostStringValidationUseCaseImpl.kt

```diff
@@ -0,0 +1,10 @@
+package com.hamdan.forzenbook.post.core.domain
+
+import com.hamdan.forzenbook.core.GlobalConstants
+
+class PostStringValidationUseCaseImpl : PostStringValidationUseCase {
```

Comment:
I'm unsure if you need a whole separate validation use case. Seems like overkill

Reply:
it does considering I turned it into a one line function.